[2018-08-13 22:45:48,239] ERROR {org.ballerinalang.net.http.HttpUtil} - call failed  
[2018-08-13 23:15:13,418] WARN  {org.wso2.transport.http.netty.sender.TargetErrorHandler} - Exception occurred in TargetHandler : An existing connection was forcibly closed by the remote host  
[2018-08-13 23:15:13,436] ERROR {org.wso2.transport.http.netty.sender.TargetErrorHandler} - Remote host closed the connection before initiating inbound response  
[2018-08-13 23:15:13,438] ERROR {org.ballerinalang.net.http.HttpUtil} - call failed  
